docs/diagrams/InfoClassDiagram.puml

```diff
@@ -0,0 +1,41 @@
+@startuml
+!include style.puml
+skinparam arrowThickness 1.1
+skinparam arrowColor UI_COLOR_T4
+skinparam classBackgroundColor UI_COLOR
+
+
+package Model <<Rectangle>>{
```

Comment:
This might not be necessary to display since class diagram shows the interactions between components and there are no interactions here to be observed. As mentioned by Sean previously, remove this unless there is an interaction.